content/en/_index.md

```diff
@@ -269,16 +269,19 @@ resource "google_project_iam_policy" "project" {
 
 ### Process the result {#result}
 
-Methods should avoid a return type whenever possible. Instead, show the user how
-to interact with a returned object programmatically by printing example
-attributes to the console. Tests should use the output of the function to ensure
-that it works.
+Depending upon the language, methods either return objects or print the value
```

Comment:
blocker: We want snippets to always print the result. Returning the result has emerged as optional, with different languages setting the circumstances of when this is done.